airflow/models/__init__.py

```diff
@@ -16,24 +16,24 @@
 # specific language governing permissions and limitations
 # under the License.
 """Airflow models"""
-from airflow.models.base import ID_LEN, Base
-from airflow.models.baseoperator import BaseOperator, BaseOperatorLink
-from airflow.models.connection import Connection
-from airflow.models.dag import DAG, DagModel, DagTag
-from airflow.models.dagbag import DagBag
-from airflow.models.dagpickle import DagPickle
-from airflow.models.dagrun import DagRun
-from airflow.models.errors import ImportError
-from airflow.models.log import Log
-from airflow.models.param import Param
-from airflow.models.pool import Pool
-from airflow.models.renderedtifields import RenderedTaskInstanceFields
-from airflow.models.sensorinstance import SensorInstance  # noqa: F401
-from airflow.models.skipmixin import SkipMixin
-from airflow.models.slamiss import SlaMiss
-from airflow.models.taskfail import TaskFail
-from airflow.models.taskinstance import TaskInstance, clear_task_instances
-from airflow.models.taskreschedule import TaskReschedule
-from airflow.models.trigger import Trigger
-from airflow.models.variable import Variable
-from airflow.models.xcom import XCOM_RETURN_KEY, XCom
+from airflow.models.base import ID_LEN, Base  # noqa: autoflake
```

Comment:
Can we ignore this whole file instead of doing it on every line?

Reply:
I could not find a way to do it. Moving # noqa: to the top of the file does not work and their doc only mention "line" exclusion

Reply:
Ahd. Found a better way !

Reply:
Autoflake has : --ignore-init-module-imports which is generally what we want !

Reply:
WAY better now.